packages/@okta/vuepress-site/docs/reference/user-query/index.md

```diff
@@ -0,0 +1,716 @@
+
+#### Request example
+
+The following example sorts all users with a status of DEPROVISIONED by last name using the default sort order (ascending):
+
+```bash
+curl -v -X GET \
+-H "Accept: application/json" \
+-H "Content-Type: application/json" \
+-H "Authorization: SSWS ${api_token}" \
+"https://${yourOktaDomain}/api/v1/users?search=status+eq+%22DEPROVISIONED%22&sortBy=profile.lastName"
+```
+
+### Array searches
+
+The `search` parameter can also search array data types for custom user profile properties. Okta follows the [SCIM Protocol Specification](https://datatracker.ietf.org/doc/html/rfc7644#section-3.4.2.2) for searching arrays. If any element of the array matches the search term, the entire array (object) is returned.
+
+You can create expressions that search multiple arrays, multiple values in an array, as well as using the standard logical and filtering operators such as starts with (`sw`), greater than (`gt`), and so on. See [Filtering](/docs/reference/core-okta-api#filter).
+
+#### Request example
+
+The following example returns all users who have a custom array value (`customProp1`) that includes at least one value equal to `a`:
+
+```bash
+curl -v -X GET \
+-H "Accept: application/json" \
+-H "Content-Type: application/json" \
+-H "Authorization: SSWS ${api_token}" \
+"https://${yourOktaDomain}/api/v1/users?search=profile.customProp1+eq+%22a%22"
+```
+
+#### Response example
+
+For brevity, only the user information and custom profile attribute are shown.
+
+```JSON
+"id": "00u3q8ta4i7sbzIQv1d7",
+        "status": "ACTIVE",
+        "created": "2022-05-20T14:41:39.000Z",
+        "activated": "2022-05-20T14:41:41.000Z",
+        "statusChanged": "2022-05-20T14:41:41.000Z",
+        "lastLogin": null,
+        "lastUpdated": "2022-05-20T14:45:47.000Z",
+        "passwordChanged": "2022-05-20T14:41:40.000Z",
+        "type": {
+            "id": "oty2di92xFgQTiKEJ1d6"
+        },
+        "profile": {
+            "firstName": "Sylvia A.",
+            "lastName": "Ray",
+            "mobilePhone": null,
+            "customProp1": [
+                "a",
+                "b",
+                "c",
+                "d",
+                "e"
+            ],
+
+```
+
+#### Request example
+
+The following example returns all users who have a custom array value (`customProp1`) that includes at least one value equal to `a` or a custom array value (`customProp2`) that includes at least one value equal to `6`:
+
+```bash
+curl -v -X GET \
+-H "Accept: application/json" \
+-H "Content-Type: application/json" \
+-H "Authorization: SSWS ${api_token}" \
+"https://${yourOktaDomain}/api/v1/users?search=profile.customProp1+eq+%22a%22+or+profile.customProp2+eq+%226%22"
```

Comment:
nit: If `customProp2` is defined to be an array of Numbers or Integers instead of an array of Strings, I believe the query would not need the quotes around `"6"`, and indeed might not match if they are present (I'm not sure about that).  No need to call this out, but illustrating it in passing might be nice.  E.g., instead of two `custom array value`s you could describe them as `a custom string array value` and `a custom number array value`, then omit the quotes around `6` in the query, and leave it at that?